ci/playbooks/edpm/run.yml

```diff
@@ -2,14 +2,19 @@
 - hosts: all
   gather_facts: true
   tasks:
```

Comment:
I missed this earlier . It should also be changed in https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci/playbooks/edpm_baremetal_deployment/run.yml as the baremetal jobs still don't use built images.

[1] https://logserver.rdoproject.org/43/243/80432abaa4078afd51ab64d80ec2a32f97fbee9a/github-check/edpm-ansible-crc-podified-edpm-baremetal/3c3771e/ci-framework-data/logs/crc/pods/openstack-operators_openstack-ansibleee-operator-controller-manager-647c77cb746tldg_30f7ca6f-621d-4287-a365-eb7039b1df42/manager/6.log